connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/RestServer.java

```diff
@@ -171,6 +172,7 @@ public void start(Herder herder) {
         resourceConfig.register(new ConnectorPluginsResource(herder));
 
         resourceConfig.register(ConnectExceptionMapper.class);
+        resourceConfig.property(ServerProperties.WADL_FEATURE_DISABLE, true);
```

Comment:
I commented out this line and `testWadlOutput` still passes.

Reply:
@hachikuji I've missed asserts. Addressed in this commit - https://github.com/apache/kafka/pull/6051/commits/f01fb9d638a64d4e3c6c8d5ba4fb8965830ac42f.